### ISSUE TITLE
Bugs in ECT60 and CIA402 drivers. Can't set some modParams

### ISSUE THREAD

**Issue by rodw-au** [linuxcnc-ethercat/linuxcnc-ethercat#438]
I am a bit stuck here. Finally tried to get homemod working.

If I use the basic_cia402 driver, I am unable to set
```
      <modParam name="peakCurrent_amps" value="2.0"/>
      <modParam name="controlMode" value="closedloop"/>	
```
6 amps into a 2 amp motor may not be wise.

If I use the ECT60 driver, I am unable to set
```
<modParam name="homeMode" value="17"/>
```
In either case, I can't play with homing further.

I might revert to the generic XML driver but I am out of time tonight.

The good news is the drive started to home!

**Comment by scottlaird** [comment]
There's a chance that homemode isn't plumbed through right for RT steppers.  I'll take a look later today.

**Comment by scottlaird** [comment]
Ah, okay -- `homeMode` isn't there anywhere.  That'd be the problem.  I'll need to add it.  I'll take a look tonight, it shouldn't be much work, but I don't have time right now.

**Comment by rodw-au** [comment]
Thanks I did look at the code but had no  idea where to start. Its good to find these things...

**Comment by scottlaird** [comment]
Yeah, the CiA spec is messy enough with almost everything being optional that the code ends up being kind of complex. It's probably only ~20 lines of code, but they'll be kind of weird. C isn't a great language for doing this sort of thing cleanly.

**Comment by grandixximo** [comment]
Fixed in 5a56ee3.

Added the `homeMethod` modParam (writes 0x6098:00 / S8 at startup), so the rotated example becomes:

```xml
<modParam name="homeMethod" value="17"/>
```

Also fixed a doc typo: `cia402.md` originally showed `homeMode` in an example, which never existed. The runtime `home-method` HAL pin still works for overriding after startup.

Note on the `peakCurrent_amps` / `controlMode` part of this report: those are RTelligent-specific modparams (`lcec_rtec.c:53,61`), they only apply to ECT/ECR drives, not to the generic `basic_cia402` driver. Using them on `basic_cia402` is expected to fail. If you want them on `basic_cia402`-based devices, that's a separate driver request.
